core-blocks/audio/edit.js

```diff
@@ -102,6 +109,37 @@ class AudioEdit extends Component {
 					</Toolbar>
 				</BlockControls>
 				<InspectorControls>
+					<PanelBody title={ __( 'Preload' ) }>
+						<ButtonGroup aria-label={ __( 'Preload' ) }>
+							{ map( [
+								{
+									name: __( 'Auto' ),
```

Comment:
I know "auto" is the HTML value for this, but we don't have to follow the HTML attributes to a T.

I think a more instructive/helpful label here would be "Content" (as that's what you're instructing the browser to preload) or, if you want more text: "Preload Content". If you really want the HTML value in brackets you could do "Content (Auto)" but I think "Content would suffice and then nix the need for a description beneath.

Reply:
I'm not particularly keen to bikeshed the labels at this point.

If you'd like to take the changes through the copy review process, then please do.